packages/framer-motion/src/motion/index.tsx

```diff
@@ -135,7 +135,7 @@ export function createMotionComponent<Props extends {}, Instance, RenderState>({
                             visualState,
                             context.visualElement,
                             externalRef
-                        ),
+                        ) as React.Ref<Instance>,
```

Comment:
Can we do without this cast? Maybe by making `useMotionRef` generic?

Reply:
Ah yeah I had to just pass a few more generics around